ebc/esfm/__init__.py

```diff
@@ -0,0 +1,33 @@
+#  /*******************************************************************************
+#   * Copyright 2022 -- 2024 EVEREST consortium (everest-h2020.eu)
+#   *
+#   * Licensed under the Apache License, Version 2.0 (the "License");
+#   * you may not use this file except in compliance with the License.
+#   * You may obtain a copy of the License at
+#   *
+#   *     http://www.apache.org/licenses/LICENSE-2.0
+#   *
+#   * Unless required by applicable law or agreed to in writing, software
+#   * distributed under the License is distributed on an "AS IS" BASIS,
+#   * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   * See the License for the specific language governing permissions and
+#   * limitations under the License.
+#  *******************************************************************************/
+#
+
+from .esfm_wrapper import EsfmWrapper
+
+module = EsfmWrapper
+identifier = 'esfm'
+
+docstrs = {'usage': f'{identifier} <pf_dev> ( init | reconf | attach | detach | list_vfs | list_vf_vm | print_kernel )',
```

Comment:
Hi @samvosys,
I think the usage string must be 
```
f"{identifier} <pf_dev> ( init <VF_NUM> | reconf <VF_NUM> | attach <VF_IDX> <VM_NAME> | detach <VF_IDX> | list_vfs | list_vf_vm | print_kernel )"
```
otherwise I can't use init, reconf etc. 
Can you change and confirm it?